2024-04-14 10:44:09,513 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2024-04-14 10:44:10,905 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50221)
2024-04-14 10:44:12,495 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50231)
2024-04-14 10:44:16,051 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '1', 'PASS': '1'}
2024-04-14 10:44:16,662 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcwNTZ9.uKg6XiOFC8BbquhMlRZ9RfcmZQoV-y9VjODfjmlz64c', 'action': 'GET_FRIEND_GROUP', 'USER': '1'}
2024-04-14 10:44:25,816 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '2', 'PASS': '2'}
2024-04-14 10:44:26,396 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjcwNjV9.zEURed-O68gWAmpTOeqFES__rZGlVO151a74AlOznB4', 'action': 'GET_FRIEND_GROUP', 'USER': '2'}
2024-04-14 10:44:33,024 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjcwNjV9.zEURed-O68gWAmpTOeqFES__rZGlVO151a74AlOznB4', 'action': 'previous', 'from': '2', 'to': '1'}
2024-04-14 10:44:34,068 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcwNTZ9.uKg6XiOFC8BbquhMlRZ9RfcmZQoV-y9VjODfjmlz64c', 'action': 'previous', 'from': '1', 'to': '2'}
2024-04-14 10:44:37,140 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcwNTZ9.uKg6XiOFC8BbquhMlRZ9RfcmZQoV-y9VjODfjmlz64c', 'action': 'message', 'time': 1713080677.1298928, 'from': '1', 'to': '2', 'message_text': 'фывфыв'}
2024-04-14 10:44:37,154 INFO ParserClientMessage.py Отправлено сообщение пользователю 2 от пользователя 1.
2024-04-14 10:44:40,707 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcwNTZ9.uKg6XiOFC8BbquhMlRZ9RfcmZQoV-y9VjODfjmlz64c', 'action': 'DELETE_MESSAGE', 'MESSAGE_ID': 110, 'USERNAME': '1'}
2024-04-14 10:44:40,717 ERROR server.py Клиент отключился от сервера. Тип исключения: AttributeError, аргументы: ("'NoneType' object has no attribute 'name'",)
2024-04-14 10:45:25,194 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2024-04-14 10:45:37,937 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50338)
2024-04-14 10:45:42,526 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '1', 'PASS': '1'}
2024-04-14 10:45:43,098 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcxNDJ9.32XadFJfvA79Xp_3n2_5oTTwsjwUlTnNOHvyy6Q3lpw', 'action': 'GET_FRIEND_GROUP', 'USER': '1'}
2024-04-14 10:45:44,626 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcxNDJ9.32XadFJfvA79Xp_3n2_5oTTwsjwUlTnNOHvyy6Q3lpw', 'action': 'previous', 'from': '1', 'to': '2'}
2024-04-14 10:45:46,667 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcxNDJ9.32XadFJfvA79Xp_3n2_5oTTwsjwUlTnNOHvyy6Q3lpw', 'action': 'DELETE_MESSAGE', 'MESSAGE_ID': 106, 'USERNAME': '1'}
2024-04-14 10:46:46,138 ERROR server.py Клиент отключился от сервера. Тип исключения: AttributeError, аргументы: ("'NoneType' object has no attribute 'name'",)
2024-04-14 10:46:46,138 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50341)
2024-04-14 10:46:46,139 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2024-04-14 10:46:49,176 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50344)
2024-04-14 10:46:51,741 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '1', 'PASS': '1'}
2024-04-14 10:46:52,324 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcyMTF9.huoqJ5OydGQXD74E2rElsD5K_dTOsrNGdXFiS5ES6Hg', 'action': 'GET_FRIEND_GROUP', 'USER': '1'}
2024-04-14 10:46:54,376 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcyMTF9.huoqJ5OydGQXD74E2rElsD5K_dTOsrNGdXFiS5ES6Hg', 'action': 'previous', 'from': '1', 'to': '2'}
2024-04-14 10:46:57,423 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcyMTF9.huoqJ5OydGQXD74E2rElsD5K_dTOsrNGdXFiS5ES6Hg', 'action': 'DELETE_MESSAGE', 'MESSAGE_ID': 105, 'USERNAME': '1'}
2024-04-14 10:47:24,145 ERROR server.py Клиент отключился от сервера. Тип исключения: AttributeError, аргументы: ("'NoneType' object has no attribute 'name'",)
2024-04-14 10:47:46,487 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2024-04-14 10:47:49,094 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50363)
2024-04-14 10:47:51,637 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '1', 'PASS': '1'}
2024-04-14 10:47:52,224 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcyNzF9.ZgWO041_KAX6hPgmhP3izcPjKLuiWjAPJSev_j_VIDo', 'action': 'GET_FRIEND_GROUP', 'USER': '1'}
2024-04-14 10:47:53,252 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcyNzF9.ZgWO041_KAX6hPgmhP3izcPjKLuiWjAPJSev_j_VIDo', 'action': 'previous', 'from': '1', 'to': '2'}
2024-04-14 10:47:58,903 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjcyNzF9.ZgWO041_KAX6hPgmhP3izcPjKLuiWjAPJSev_j_VIDo', 'action': 'DELETE_MESSAGE', 'MESSAGE_ID': 104, 'USERNAME': '1'}
2024-04-14 10:50:51,704 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2024-04-14 10:50:57,487 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50375)
2024-04-14 10:51:03,599 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '1', 'PASS': '1'}
2024-04-14 10:51:04,187 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc0NjN9.aklkPn_IlmMc8oZHhMRhTrQ60O-LWxu2tuzXgXilMj8', 'action': 'GET_FRIEND_GROUP', 'USER': '1'}
2024-04-14 10:51:05,720 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc0NjN9.aklkPn_IlmMc8oZHhMRhTrQ60O-LWxu2tuzXgXilMj8', 'action': 'previous', 'from': '1', 'to': '2'}
2024-04-14 10:51:08,777 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc0NjN9.aklkPn_IlmMc8oZHhMRhTrQ60O-LWxu2tuzXgXilMj8', 'action': 'DELETE_MESSAGE', 'MESSAGE_ID': 103, 'USERNAME': '1'}
2024-04-14 10:51:19,011 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc0NjN9.aklkPn_IlmMc8oZHhMRhTrQ60O-LWxu2tuzXgXilMj8', 'action': 'DELETE_MESSAGE', 'MESSAGE_ID': 102, 'USERNAME': '1'}
2024-04-14 10:51:25,924 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50380)
2024-04-14 10:51:28,440 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '2', 'PASS': '2'}
2024-04-14 10:51:29,019 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc0ODh9.yirdGJ6TshfChU9wN0Ivk2M8mvRKQojCbyot9PDgCK8', 'action': 'GET_FRIEND_GROUP', 'USER': '2'}
2024-04-14 10:51:33,618 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc0ODh9.yirdGJ6TshfChU9wN0Ivk2M8mvRKQojCbyot9PDgCK8', 'action': 'previous', 'from': '2', 'to': '1'}
2024-04-14 10:51:39,763 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc0ODh9.yirdGJ6TshfChU9wN0Ivk2M8mvRKQojCbyot9PDgCK8', 'action': 'message', 'time': 1713081099.521285, 'from': '2', 'to': '1', 'message_text': 'hiiii'}
2024-04-14 10:51:39,773 INFO ParserClientMessage.py Отправлено сообщение пользователю 1 от пользователя 2.
2024-04-14 10:51:45,865 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc0ODh9.yirdGJ6TshfChU9wN0Ivk2M8mvRKQojCbyot9PDgCK8', 'action': 'UPDATE_MESSAGE', 'MESSAGE_ID': 111, 'UPDATE_TEXT': 'qazwsx', 'USERNAME': '2'}
2024-04-14 10:52:20,540 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc0NjN9.aklkPn_IlmMc8oZHhMRhTrQ60O-LWxu2tuzXgXilMj8', 'action': 'previous', 'from': '1', 'to': '2'}
2024-04-14 10:52:25,670 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc0NjN9.aklkPn_IlmMc8oZHhMRhTrQ60O-LWxu2tuzXgXilMj8', 'action': 'SEARCH_MESSAGE', 'SEARCH_TEXT': 'qa', 'USERNAME': '1', 'GROUP': None, 'TO': '2'}
2024-04-14 10:52:37,862 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc0ODh9.yirdGJ6TshfChU9wN0Ivk2M8mvRKQojCbyot9PDgCK8', 'action': 'SEARCH_MESSAGE', 'SEARCH_TEXT': 'qa', 'USERNAME': '2', 'GROUP': None, 'TO': '1'}
2024-04-14 10:52:42,962 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc0NjN9.aklkPn_IlmMc8oZHhMRhTrQ60O-LWxu2tuzXgXilMj8', 'action': 'SEARCH_MESSAGE', 'SEARCH_TEXT': 'qa', 'USERNAME': '1', 'GROUP': None, 'TO': '2'}
2024-04-14 10:53:58,908 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2024-04-14 10:54:01,823 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50392)
2024-04-14 10:54:04,879 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '1', 'PASS': '1'}
2024-04-14 10:54:05,471 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc2NDR9.DSXR07Ik0-CndV6XOXhXhPVKLOKvzloXy05JcsN8B0I', 'action': 'GET_FRIEND_GROUP', 'USER': '1'}
2024-04-14 10:54:06,498 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc2NDR9.DSXR07Ik0-CndV6XOXhXhPVKLOKvzloXy05JcsN8B0I', 'action': 'previous', 'from': '1', 'to': '2'}
2024-04-14 10:54:09,563 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc2NDR9.DSXR07Ik0-CndV6XOXhXhPVKLOKvzloXy05JcsN8B0I', 'action': 'SEARCH_MESSAGE', 'SEARCH_TEXT': 'qa', 'USERNAME': '1', 'GROUP': None, 'TO': '2'}
2024-04-14 10:54:20,109 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50395)
2024-04-14 10:54:24,213 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '2', 'PASS': '2'}
2024-04-14 10:54:24,791 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc2NjR9.GaJAsyATFDxFS10B_pVRrUrUynSqbukMjv4oXZl8qCo', 'action': 'GET_FRIEND_GROUP', 'USER': '2'}
2024-04-14 10:54:32,422 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc2NjR9.GaJAsyATFDxFS10B_pVRrUrUynSqbukMjv4oXZl8qCo', 'action': 'previous', 'from': '2', 'to': '1'}
2024-04-14 10:54:35,505 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc2NjR9.GaJAsyATFDxFS10B_pVRrUrUynSqbukMjv4oXZl8qCo', 'action': 'message', 'time': 1713081275.5010622, 'from': '2', 'to': '1', 'message_text': 'sdfsdfsdf'}
2024-04-14 10:54:35,515 INFO ParserClientMessage.py Отправлено сообщение пользователю 1 от пользователя 2.
2024-04-14 10:54:38,548 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc2NjR9.GaJAsyATFDxFS10B_pVRrUrUynSqbukMjv4oXZl8qCo', 'action': 'DELETE_MESSAGE', 'MESSAGE_ID': 112, 'USERNAME': '2'}
2024-04-14 10:56:36,306 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2024-04-14 10:56:36,306 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2024-04-14 10:56:36,854 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50402)
2024-04-14 10:56:39,903 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '1', 'PASS': '1'}
2024-04-14 10:56:40,476 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc3OTl9.Rivw075OYamZpQSBXD3-lszd4cbBgfdUPSAoXwYYRb8', 'action': 'GET_FRIEND_GROUP', 'USER': '1'}
2024-04-14 10:56:43,014 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50406)
2024-04-14 10:56:46,076 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '2', 'PASS': '2'}
2024-04-14 10:56:46,653 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc4MDZ9.AC8dv-w7HWUTFarcUO7bU6Go1q3thSf4SuEa69qXVkk', 'action': 'GET_FRIEND_GROUP', 'USER': '2'}
2024-04-14 10:56:51,251 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc3OTl9.Rivw075OYamZpQSBXD3-lszd4cbBgfdUPSAoXwYYRb8', 'action': 'previous', 'from': '1', 'to': '2'}
2024-04-14 10:56:52,282 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc4MDZ9.AC8dv-w7HWUTFarcUO7bU6Go1q3thSf4SuEa69qXVkk', 'action': 'previous', 'from': '2', 'to': '1'}
2024-04-14 10:56:57,900 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc4MDZ9.AC8dv-w7HWUTFarcUO7bU6Go1q3thSf4SuEa69qXVkk', 'action': 'DELETE_MESSAGE', 'MESSAGE_ID': 111, 'USERNAME': '2'}
2024-04-14 10:57:00,968 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc3OTl9.Rivw075OYamZpQSBXD3-lszd4cbBgfdUPSAoXwYYRb8', 'action': 'DELETE_MESSAGE', 'MESSAGE_ID': 101, 'USERNAME': '1'}
2024-04-14 10:57:04,038 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc3OTl9.Rivw075OYamZpQSBXD3-lszd4cbBgfdUPSAoXwYYRb8', 'action': 'message', 'time': 1713081423.6413138, 'from': '1', 'to': '2', 'message_text': 'adasdasdasdasd'}
2024-04-14 10:57:04,049 INFO ParserClientMessage.py Отправлено сообщение пользователю 2 от пользователя 1.
2024-04-14 10:57:07,095 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc4MDZ9.AC8dv-w7HWUTFarcUO7bU6Go1q3thSf4SuEa69qXVkk', 'action': 'message', 'time': 1713081427.075954, 'from': '2', 'to': '1', 'message_text': 'sadasdasdasd'}
2024-04-14 10:57:07,101 INFO ParserClientMessage.py Отправлено сообщение пользователю 1 от пользователя 2.
2024-04-14 10:57:09,668 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc3OTl9.Rivw075OYamZpQSBXD3-lszd4cbBgfdUPSAoXwYYRb8', 'action': 'message', 'time': 1713081429.5951738, 'from': '1', 'to': '2', 'message_text': 'qqqqq'}
2024-04-14 10:57:09,674 INFO ParserClientMessage.py Отправлено сообщение пользователю 2 от пользователя 1.
2024-04-14 10:57:11,707 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc4MDZ9.AC8dv-w7HWUTFarcUO7bU6Go1q3thSf4SuEa69qXVkk', 'action': 'message', 'time': 1713081431.56063, 'from': '2', 'to': '1', 'message_text': 'aaaaaa'}
2024-04-14 10:57:11,714 INFO ParserClientMessage.py Отправлено сообщение пользователю 1 от пользователя 2.
2024-04-14 10:57:15,264 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc4MDZ9.AC8dv-w7HWUTFarcUO7bU6Go1q3thSf4SuEa69qXVkk', 'action': 'DELETE_MESSAGE', 'MESSAGE_ID': 114, 'USERNAME': '2'}
2024-04-14 10:57:17,292 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc3OTl9.Rivw075OYamZpQSBXD3-lszd4cbBgfdUPSAoXwYYRb8', 'action': 'DELETE_MESSAGE', 'MESSAGE_ID': 115, 'USERNAME': '1'}
2024-04-14 10:57:26,493 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc3OTl9.Rivw075OYamZpQSBXD3-lszd4cbBgfdUPSAoXwYYRb8', 'action': 'UPDATE_MESSAGE', 'MESSAGE_ID': 113, 'UPDATE_TEXT': 'changed', 'USERNAME': '1'}
2024-04-14 10:57:46,282 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc4MDZ9.AC8dv-w7HWUTFarcUO7bU6Go1q3thSf4SuEa69qXVkk', 'action': 'previous', 'from': '2', 'to': '1'}
2024-04-14 10:58:20,936 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2024-04-14 10:58:20,936 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2024-04-14 10:58:21,796 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50417)
2024-04-14 10:58:21,885 INFO server.py Установлено соедение с клиентом ('127.0.0.1', 50418)
2024-04-14 10:58:24,441 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '1', 'PASS': '1'}
2024-04-14 10:58:25,017 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc5MDR9.SyVu5gKRsJi-wRJ98hK17Xp8nRLha1xfEXZLLHckbxg', 'action': 'GET_FRIEND_GROUP', 'USER': '1'}
2024-04-14 10:58:29,087 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'action': 'LOGIN', 'USERNAME': '2', 'PASS': '2'}
2024-04-14 10:58:29,664 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc5MDl9.6ca1YHJmsVP-DDtUWPOQiMDdls2hobq-moOJoCmupV8', 'action': 'GET_FRIEND_GROUP', 'USER': '2'}
2024-04-14 10:58:30,694 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjIiLCJwYXNzd29yZCI6IjIiLCJleHAiOjE3MTMxNjc5MDl9.6ca1YHJmsVP-DDtUWPOQiMDdls2hobq-moOJoCmupV8', 'action': 'previous', 'from': '2', 'to': '1'}
2024-04-14 10:58:32,228 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc5MDR9.SyVu5gKRsJi-wRJ98hK17Xp8nRLha1xfEXZLLHckbxg', 'action': 'previous', 'from': '1', 'to': '2'}
2024-04-14 10:58:46,490 DEBUG ParserClientMessage.py Разбор сообщения от клиента: {'TOKEN': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6IjEiLCJwYXNzd29yZCI6IjEiLCJleHAiOjE3MTMxNjc5MDR9.SyVu5gKRsJi-wRJ98hK17Xp8nRLha1xfEXZLLHckbxg', 'action': 'UPDATE_MESSAGE', 'MESSAGE_ID': 113, 'UPDATE_TEXT': 'qaz', 'USERNAME': '1'}
2024-04-14 10:59:05,812 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2024-04-14 10:59:06,819 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2024-04-14 10:59:11,452 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
